docs/modules/ROOT/pages/runbooks/resourcerequests.adoc

```diff
@@ -0,0 +1,205 @@
+= Alert Group: syn-ResourceRequests
+
+We provide four rules to notice if a cluster is hitting the limit of requested resources.
+If a cluster has too many requested resources it might not be able to schedule all workloads.
+
+== Alert Rule: SYN_TooMuchMemoryRequested [[SYN_TooMuchMemoryRequested]]
+
+=== icon:glasses[] Overview
+
+This alert means that the total sum of memory requests of user workload is close to the capacity of the cluster.
+By default it will fire if the amount of unrequested memory is less than the capacity of the largest worker node.
+
+This means if you receive this alert the cluster likely won’t be able to keep all workloads running in case of a worker node failure.
+After verifying the alert you should add more memory capacity as soon as possible.
+
+=== icon:search[] Investigate
+
+* Verify the alert
+** Verify that the reported capacity is correct.
++
+[source,shell]
+----
+kubectl describe node -lnode-role.kubernetes.io/app | grep "Allocated resources" -A 8
+----
++
+If not there might be a bug in the alert rule.
+Please disable this alert and open an issue for this component.
+** Check if there is a sudden increase in resource requests that indicate that this might be temporary.
+* Either add more worker nodes or resize existing worker nodes according to the install https://kb.vshn.ch/oc4/index.html[instructions for your cloud]
+
+=== icon:wrench[] Tune
+
+If this alert is non actionable, noisy, or was raised too late you might want to tune it.
+
+Through the component parameters you have the option modify the alert threshold, change for how long it needs to be firing until you are alerted, or disable it outright.
+In the example below will adapt the rule to only alert after two hours and change the threshold to 128 GB.
+
+[source,yaml]
+----
+capacityAlerts:
+  groups:
+    ResourceRequests:
+      rules:
+        TooMuchMemoryRequested:
+          enabled: true
+          for: 2h
+          expr:
+            threshold: '128*1024*1024*1024' <1>
+----
+<1> The threshold can be an arbitrary promql expression
+
+== Alert Rule: SYN_ExpectTooMuchMemoryRequested [[SYN_ExpectTooMuchMemoryRequested]]
+
+=== icon:glasses[] Overview
+
+This alert means that we expect the total sum of memory requests of user workload to soon be close to the capacity of the cluster.
+By default it will fire if the predicted amount of unrequested memory in 3 days is less than the capacity of the largest worker node.
```

Comment:
```suggestion
By default, it will fire if the predicted amount of unrequested memory in three days is less than the capacity of the largest worker node.
```